ui/components/subscription/CitizenMetadataModal.tsx

```diff
@@ -124,21 +197,54 @@ export function CitizenMetadataModal({ nft, selectedChain, setEnabled }: any) {
         )}
         {stage === 1 && (
           <>
-            <Widget
-              className="w-[100%] md:w-[100%]"
-              id={process.env.NEXT_PUBLIC_TYPEFORM_CITIZEN_FORM_ID as string}
-              onSubmit={submitTypeform}
-              height={500}
-            />
+            <p>Would you like to update your email?</p>
```

Comment:
Stages works here - but I wonder if you could just surface all of them togther at the same time?

So clicking the "Edit" icon would pop up a modal with three options:
1.) Edit Image
2.) Edit Email
3.) Edit Profile

And then clicking on one lets you go to it directly, versus having to click through stages each time?

I think that would be more intuitive, but it might be a pain to implement. If it ends up being a big restructure just ignore, but if it's easy to do then that could be better